tasks/uk/census/metadata.py

```diff
@@ -0,0 +1,82 @@
+from collections import OrderedDict
+import json
+import os
+import re
+
+from tasks.meta import OBSColumn, DENOMINATOR, OBSTag
+from tasks.util import ColumnsTask, TagsTask
+from tasks.tags import UnitTags, SectionTags, SubsectionTags, LicenseTags
+
+
+def load_definition():
+    with open(os.path.join(os.path.dirname(__file__), 'columns.json')) as json_file:
+        return json.load(json_file, object_pairs_hook=OrderedDict)
+
+
+COLUMNS_DEFINITION = load_definition()
+
+
+def parse_table(table_id):
+    '''
+    Returns a tuple like (KS, NI) from a table string like KS201NI
+    '''
+    return re.match('([A-Z]{2})\d+([A-Z]{2})[a-z]?', table_id).groups()
+
+
+class SourceTags(TagsTask):
+    def version(self):
+        return 1
+
+    def tags(self):
+        return [OBSTag(id='ons',
+                       name='Office for National Statistics (ONS)',
+                       type='source',
+                       description="The UK's largest independent producer of official statistics and the recognised national statistical institute of the UK (`ONS <https://www.ons.gov.uk/>`_)"),
+                OBSTag(id='scotland-census',
+                       name="Scotland's Census Data Warehouse by National Records of Scotland",
+                       type='source',
+                       description="`Scotland Census <http://www.scotlandscensus.gov.uk/`_"),
+                OBSTag(id='nisra',
+                       name="Northern Ireland Statistics and Research Agency",
+                       type='source',
+                       description="`Northern Ireland Statistics and Research Agency <https://www.nisra.gov.uk/`_")]
+
+
+class CensusColumns(ColumnsTask):
+    def requires(self):
+        return {
+            'units': UnitTags(),
+            'sections': SectionTags(),
+            'subsections': SubsectionTags(),
+            'source': SourceTags(),
+            'license': LicenseTags(),
+        }
+
+    def version(self):
+        return 5
+
+    def prefix(self):
+        return 'uk.ons'
+
+    def columns(self):
+        input_ = self.input()
+        sources = input_['source'].values()
+        license = input_['license']['uk_ogl']
```

Comment:
We should start changing license for other name because in Python `license` is a reserved word

Reply:
Uhm, yeah, we can make this one be the first one :)